src/test/scala/no/ndla/draftapi/service/ConverterServiceTest.scala

```diff
@@ -141,36 +141,29 @@ class ConverterServiceTest extends UnitSuite with TestEnvironment {
     res.forall { case (_, to) => to.isEmpty } should be(true)
   }
 
-  test("stateTransitionsToApi should return only certain entries if user only has write roles") {
-    val res = service.stateTransitionsToApi(TestData.userWithWriteAccess)
-    res(IMPORTED.toString).length should be(1)
-    res(DRAFT.toString).length should be(2)
-    res(PROPOSAL.toString).length should be(5)
-    res(USER_TEST.toString).length should be(4)
-    res(AWAITING_QUALITY_ASSURANCE.toString).length should be(6)
-    res(QUALITY_ASSURED.toString).length should be(2)
-    res(QUALITY_ASSURED_DELAYED.toString).length should be(2)
-    res(QUEUED_FOR_PUBLISHING.toString).length should be(2)
-    res(QUEUED_FOR_PUBLISHING_DELAYED.toString).length should be(2)
-    res(PUBLISHED.toString).length should be(2)
-    res(AWAITING_UNPUBLISHING.toString).length should be(2)
-    res(UNPUBLISHED.toString).length should be(3)
-  }
-
-  test("stateTransitionsToApi should return all entries if user is admin") {
-    val res = service.stateTransitionsToApi(TestData.userWithAdminAccess)
-    res(IMPORTED.toString).length should be(1)
-    res(DRAFT.toString).length should be(4)
-    res(PROPOSAL.toString).length should be(9)
-    res(USER_TEST.toString).length should be(6)
-    res(AWAITING_QUALITY_ASSURANCE.toString).length should be(8)
-    res(QUALITY_ASSURED.toString).length should be(6)
-    res(QUALITY_ASSURED_DELAYED.toString).length should be(4)
-    res(QUEUED_FOR_PUBLISHING.toString).length should be(4)
-    res(QUEUED_FOR_PUBLISHING_DELAYED.toString).length should be(4)
-    res(PUBLISHED.toString).length should be(5)
-    res(AWAITING_UNPUBLISHING.toString).length should be(5)
-    res(UNPUBLISHED.toString).length should be(5)
+  test("stateTransitionsToApi should return different number of transitions based on access") {
```

Comment:
Denne testen vil være litt mindre streng nå, men kanskje verdt det for å slippe å oppdatere den hele tiden?

Reply:
Det var tanken. Har irritert meg at den var så streng, og ser heller ikkje heilt gevinsten. 
Eg vurderte å legge til en test som sjekka antall overganger for en status og så sammenligne med resultatet av service-kallet, men fant ut at testkoden ville bli identisk med det som skjer i servicen så gevinsten ville være minimal.